substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/HasGetTotalMemorySize0.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, 2022, Oracle and/or its affiliates. All rights reserved.
```

Reply:
Thanks, fixed. Pesky style issues ;-)

Reply:
Unfortunately it's still not happy:

```
/home/runner/work/graal/graal/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/CgroupMetricsJDK.java:32: 'static' modifier out of order with the JLS suggestions.
```

Reply:
Yes, trying with `static final` over `final static`